DataSourceVMwareGuestInfo.py

```diff
@@ -326,15 +321,15 @@ def get_guestinfo_value(key):
 
     if data_access_method == VMWARE_RPCTOOL:
         try:
-            (stdout, stderr) = subp_module.subp(
+            (stdout, stderr) = subp.subp(
```

Comment:
You have to use the `subp` method directly without the module, e.g `(stdout, stderr) = subp(`
Same for the other calls.

Reply:
Good point. Fixed it.